source/main/services/config.ts

```diff
@@ -89,3 +90,34 @@ export async function setVaultSettings(
     const storage = getVaultSettingsStorage(sourceID);
     await storage.setValues(settings);
 }
+
+export async function getStartInBackground(): Promise<boolean> {
+    const storage = getConfigStorage();
+    const preferences = await storage.getValue("preferences");
+    if (typeof preferences === "undefined") {
+        return false;
+    }
+    return preferences.startInBackground;
+}
+
+export async function setStartWithSession(enable: boolean): Promise<void> {
+    var AutoLaunch = require("auto-launch");
```

Comment:
Why the require here? Can this just be imported?

Also, var => const..

Reply:
This is fixed by https://github.com/Oxalin/buttercup-desktop/commit/fd7ff2e9ed0da0bfa34ea124ab1eb122383dd6c9